src/main.lua

```diff
@@ -2256,7 +2285,7 @@ addEventingHandler("auctionPrices", utils.hasMatchingTag("Action", ActionMap.Auc
 	local name = string.lower(msg.Tags.Name)
 	local auction = arns.getAuction(name)
 	local timestamp = tonumber(msg.Tags.Timestamp or msg.Timestamp)
-	local type = msg.Tags.Type or "permabuy"
+	local type = msg.Tags["Purchase-Type"] or "permabuy"
```

Comment:
Should there be an analogous change in a test somewhere?

Reply:
the current integration test validates the defaults. will add another for providing one

Reply:
i'll add a test with different tags for lease expiration ones to validate these